blivet/dbus/blivet.py

```diff
@@ -32,21 +31,24 @@
 from .object import DBusObject
 
 
+def sorted_object_paths_from_list(obj_list):
+    devices = sorted(obj_list, key=lambda d: d.object_path)
+    devices.sort(key=lambda d: d.id)
+    return [d.object_path for d in devices]
```

Comment:
I think this shouldn't refer to "devices" if it's supposed to be a generic function. If not, it should be named like ``sorted_devices_from_list``.